plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridUser.java

```diff
@@ -115,4 +157,140 @@ public List<CubridView> getSystemViews(DBRProgressMonitor monitor) throws DBExce
         }
         return views;
     }
+
+    @Nullable
+    @Override
+    public List<GenericTableIndex> getIndexes(@NotNull DBRProgressMonitor monitor) throws DBException {
+        cacheIndexes(monitor);
+        return cubridIndexCache.getAllObjects(monitor, this);
+    }
+
+    private void cacheIndexes(@NotNull DBRProgressMonitor monitor) throws DBException {
+        synchronized (cubridIndexCache) {

```

Comment:
Looks strange.
Maybe we don't need cache in class variable? Just collect table indexes on every getIndexes call?

Reply:
Thanks for suggestion. I have already removed the cache.